scripts/install.sh

```diff
@@ -71,8 +71,12 @@ create_directory() {
 }
 
 check_sudo() {
-    timeout 2 sudo id && sudo=1 || sudo=0
-    return $sudo
+    if [[ $EUID -eq 0 ]]; then
+        # The script is invoked with root permission
```

Comment:
This comment is still incorrect - the function is checking whether the current user has administrative privileges.